src/openrct2/ride/ride.c

```diff
@@ -5480,7 +5480,7 @@ sint32 ride_get_refund_price(sint32 ride_id)
         if (type != TRACK_ELEM_INVERTED_90_DEG_UP_TO_FLAT_QUARTER_LOOP){
             money32 removePrice = game_do_command(
                 x,
-                GAME_COMMAND_FLAG_APPLY | (rotation << 8),
+                GAME_COMMAND_FLAG_5 | GAME_COMMAND_FLAG_APPLY | (rotation << 8),
```

Comment:
Do we know enough about flag 5 to name it?

Reply:
I think its mostly used for previews.